WordPress/Classes/Utility/BuildInformation/FeatureFlagOverrideStore.swift

```diff
@@ -46,10 +46,17 @@ struct FeatureFlagOverrideStore {
     /// If no override exists, returns `nil`.
     ///
     func overriddenValue(for featureFlag: OverridableFlag) -> Bool? {
-        guard store.object(forKey: featureFlag.key) != nil else {
+        guard let object = store.object(forKey: featureFlag.key) else {
+            return nil
+        }
+        switch object {
+        case let value as Bool:
+            return value
+        case let value as String:
+            return (value as NSString).boolValue
```

Comment:
The way we set these in UI tests is with the launch arguments that support only strings `[String]`:

```
"-ff-override-New Stats", "true",
```

There was a recent regression in `overriddenValue` where it was changed to support only `Bool` value – incompatible with string launch arguments.